specification/scvmm/resource-manager/readme.ruby.md

```diff
@@ -4,7 +4,7 @@ These settings apply only when `--ruby` is specified on the command line.
 
 ```yaml
 package-name: azure_mgmt_scvmm
-package-version: 2020-06-05-preview
```

Comment:
Please remove this file, ruby is no longer supported.

Reply:
@lirenhe 
Removed **readme.ruby.md**.